src/Controller/CourseListController.php

```diff
@@ -61,7 +61,10 @@ public function build() {
 
     $normalizedCourses = [];
     foreach ($courses as $course) {
-      $normalizedCourses[] = $this->normalizeNode($course);
+      $normalizedCourse = $this->normalizeNode($course);
```

Comment:
How come they are were loaded but not accessible? Should be resolved before normalizing it IMO

Reply:
@ikorgik

Reply:
Added @todo to include access check when we will be configuring global permissions. Don't want to spend time on it now

Reply:
@spleshka